packages/atomic/project.json

```diff
@@ -1 +1,264 @@
-{}
+{
+  "namedInputs": {
+    "tests": ["{projectRoot}/src/**/*.{spec.{ts,tsx},spec.{ts,tsx}.snap}"],
+    "e2e": ["{projectRoot}/cypress/**"],
+    "stories": ["{projectRoot}/src/**/*.{?(*([^.]).)stories.{tsx,css},mdx}"],
+    "source": [
+      "{projectRoot}/stencil.config.ts",
+      "{projectRoot}/collection-manifest.json",
+      "{projectRoot}/tailwind.config.js",
+      "{projectRoot}/svg.transform.js",
+      "{workspaceRoot}/tsconfig.json",
+      "{projectRoot}/tsconfig.json",
+      "{projectRoot}/src/**/*([^.]).!({{spec.{ts,tsx},spec.{ts,tsx}.snap},{?(*([^.]).)stories.{tsx,css},mdx}})",
+      "{projectRoot}/stencil-plugin/**"
+    ]
+  },
+  "targets": {
+    "clean": {
+      "dependsOn": [],
+      "inputs": [],
+      "outputs": [],
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "rimraf dist docs loader www storybook-static src/components/search/atomic-search-interface/lang src/generated",
+        "cwd": "packages/atomic"
+      }
+    },
+    "cached:build:locales": {
+      "dependsOn": [],
+      "inputs": [
+        "{projectRoot}/scripts/create-generated-folder.mjs",
+        "{projectRoot}/scripts/split-locales.mjs",
+        "{projectRoot}/scripts/copy-dayjs-locales.mjs",
+        "{projectRoot}/src/locales.json"
+      ],
+      "outputs": [
+        "{projectRoot}/src/generated/availableLocales.json",
+        "{projectRoot}/src/generated/dayjs-locales-data.ts",
+        "{projectRoot}/src/components/search/atomic-search-interface/lang"
+      ],
+      "executor": "nx:run-commands",
+      "options": {
+        "commands": [
+          "node ./scripts/create-generated-folder.mjs",
+          "node ./scripts/split-locales.mjs",
+          "node ./scripts/copy-dayjs-locales.mjs"
+        ],
+        "parallel": false,
+        "cwd": "packages/atomic"
+      }
+    },
+    "cached:build:stencil": {
+      "dependsOn": ["^build", "cached:build:locales"],
+      "inputs": ["^source", "source"],
+      "outputs": [
+        "{projectRoot}/dist",
+        "{projectRoot}/www",
+        "{projectRoot}/docs",
+        "{projectRoot}/loader",
+        "{projectRoot}/src/components.d.ts"
+      ],
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "node --max_old_space_size=6144 ../../node_modules/@stencil/core/bin/stencil build",
+        "cwd": "packages/atomic"
+      }
+    },
+    "cached:validate:definitions": {
+      "dependsOn": ["^build", "cached:build:stencil"],
+      "inputs": ["{projectRoot}/dist/types/components.d.ts"],
+      "outputs": [],
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "tsc --noEmit --esModuleInterop --skipLibCheck ./dist/types/components.d.ts",
```

Comment:
There's an official `tsc` executor (see [`@nrwl/js:tsc`](https://nx.dev/packages/js/executors/tsc#@nrwl/js:tsc)), but its configuration is quite different from what we're using at the moment, so I preferred to stick to the command line.